pkg/apis/duck/v1alpha1/subscribable_types_conversion.go

```diff
@@ -32,11 +32,16 @@ func (source *SubscribableType) ConvertTo(ctx context.Context, obj apis.Converti
 	switch sink := obj.(type) {
 	case *duckv1beta1.Subscribable:
```

Comment:
I am gonna finally add v1alpha1<>v1 conversions after this PR. Please bear with me.